ppr-ui/src/components/tables/mhr/SearchedResults.vue

```diff
@@ -40,7 +41,13 @@
       </v-row>
       <v-row v-else class="result-info">
         <v-col id="review-results-count" cols="auto">
-          <span><b>{{ selectedMatchesLength }}</b> Manufactured Homes</span>
+          <span>Matches Selected: <b>{{ selectedMatchesLength }}</b></span>
+        </v-col>
+        <v-col id="review-registrations-count" cols="auto">
+          <span>Registrations:  <b>{{ uniqueResults.length }}</b></span>
```

Comment:
What does `uniqueResults.length` resolve too if uniqueResults is null or undefined?

Reply:
Im not sure, but can you think of a case when uniqueResults is undefined/null? It initializes as null but is always set before use with lodashs uniqBy. If no unique selections were found it will return an empty array, meaning length of 0. Let me know if this clears things up

Reply:
It does, sort of :) 

This is not a deal breaker and i purely asked because the initialized value (`null`) as you mention would result in a type error here if you tried to render this property accessor (`Cannot read property 'length' of null`). 

In script, we can often protect ourselves from timing issues using optional chaining (`uniqueResults?.length`) but this isn't recognized in the template. 

tldr: In scenarios like this, it might be wise to initialize your property with it's intended type, to protect your template references from typeError. ie `uniqueResults = []` if safe to do so. 

IF it was initialized as `null` explicitly for some type of truthy check, you could still reference `!!localState.uniqueResults` as afaik an empty array would be falsy.

Reply:
I see, I'll check with Dima if there was a purpose for init as null instead of [] as this was part of his approach

Reply:
Resolving the comment, this was more for awareness. Clearly it's not breaking anything 😁 
Just wanted to highlight the importance of initialization and property types and all that fun stuff 😆